src/main/java/com/example/demo/Models/MovieCatalog.java

```diff
@@ -0,0 +1,47 @@
+package com.example.demo.Models;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.OneToOne;
+import javax.persistence.JoinColumn;
+
+@Data
+@Entity
+public class MovieCatalog {
+
+    @Id
+    @Column(name = "movieID")
+    private Long movieID;
+
+    @OneToOne(cascade = CascadeType.MERGE)
```

Comment:
The different relation types have a property: fetch, that property could have different values, could you explain please what does they mean?